Dockerfile

```diff
@@ -13,4 +13,7 @@ WORKDIR /app
 ENV NODE_ENV production
 RUN yarn install --frozen-lockfile
 
+COPY docker-entrypoint.sh /usr/local/bin/
+ENTRYPOINT ["docker-entrypoint.sh"]
```

Comment:
Why not just `ENTRYPOINT ["node", "index.js"]` and then we pass in the `-s ...` extra arguments?

Reply:
what if you want to control the docker container itself and not dockly specifically, the `docker-entrypoint.sh` script enables that

Reply:
when you say control you mean to spawn a shell for debugging purposes? if so, yeah you'd need to rebuild the image without the entrypoint, which I think is ok to do as it's not something we'd need to do constantly and easier to maintain. WDYT?

Reply:
Ok, I agree. I'll remove it